fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -403,6 +403,8 @@ int getSrcWorkFactor(RelocateData const& relocation, int singleRegionTeamSize) {
 		return WORK_FULL_UTILIZATION / SERVER_KNOBS->RELOCATION_PARALLELISM_PER_SOURCE_SERVER;
 	else if (relocation.healthPriority == SERVER_KNOBS->PRIORITY_TEAM_2_LEFT)
 		return WORK_FULL_UTILIZATION / 2 / SERVER_KNOBS->RELOCATION_PARALLELISM_PER_SOURCE_SERVER;
+	else if (relocation.healthPriority == SERVER_KNOBS->PRIORITY_PERPETUAL_STORAGE_WIGGLE)
```

Comment:
Does it require setting a higher priority for perpetual wiggle?

Reply:
Yes. Otherwise, the priority here is PRIORITY_TEAM_HEALTHY.

Reply:
Please add comments and update the knob accordingly.